dev/package-vcpkg.sh

```diff
@@ -13,5 +16,13 @@ if [ "$LINUX_OS" == "centos" ]; then
     source /opt/rh/devtoolset-9/enable
   fi
 fi
+
+# prepare vcpkg environments
 source ./dev/vcpkg/env.sh
-./dev/buildbundle-veloxbe.sh --build_tests=ON --build_benchmarks=ON --enable_s3=ON  --enable_hdfs=ON
+
+# build gluten with velox backend, prompt always respond y
+export PROMPT_ALWAYS_RESPOND=y
+./dev/buildbundle-veloxbe.sh --build_tests=ON --build_benchmarks=ON --enable_s3=ON  --enable_hdfs=ON "$@"
+
+# make thirdparty package
+./dev/build-thirdparty.sh
```

Comment:
This script is for vcpkg build with all dependencies statically linked. No need to pack shared libs.

Reply:
> This script is for vcpkg build with all dependencies statically linked. No need to pack shared libs.

Sorry, I didn't look at the document carefully, does it mean we no longer need the thirdparty lib jar when using `package-vcpkg.sh`? I will try it, thank you.

Reply:
@wForget, yes, third-party jar is useless when vcpkg build is enabled.

Reply:
Thank you, I will close this PR.

Reply:
> Extract build-thirdparty.sh script
> Set PROMPT_ALWAYS_RESPOND=y for package scripts
> Pass package arguments to buildbundle-veloxbe.sh

Are these improvements reasonable? If so, I will continue working

Reply:
@wForget , yes, you can just keep these changes. Thanks!